src/status_im/contexts/profile/profiles/view.cljs

```diff
@@ -273,7 +277,7 @@
          :icon-left           :i/unlocked
          :disabled?           (or (not sign-in-enabled?) processing)
          :on-press            login-profile
-         :container-style     {:margin-bottom (+ (safe-area/get-bottom) 12)}}
+         :container-style     {:margin-bottom (+ bottom 12)}}
         (i18n/label :t/log-in)])]))
```

Comment:
Here's another spot where we can try to hide the value `12`, do you think that's possible or preferred?

Reply:
Oh, that `12` is because of designs, we add a padding of 12 so that the list has an empty space at the end instead of snapping to the safe area. this is just to match designs